examples/get-started/first_figure.py

```diff
@@ -91,16 +91,19 @@
 # Set the projection
 # ------------------
 #
-# This figure now has its colors set, but there is no projection or size
-# set for the map. Both of these values are set using the ``projection``
-# parameter.
+# This figure now has its colors set. By default the projection and size
+# of the map is set to ``"Q15c+du"``. Here, ``"Q15c"`` defines that a
+# cylindrical equidistant map projection with a width of 15 cm is used
+# and ``"+du"`` selects the minimum map dimension. However, both of
```

Comment:
> ``"Q15c"`` defines that a cylindrical equidistant map projection with a width of 15 cm is used and ``"+du"`` selects the minimum map dimension.

This sentence needs some improvements. `Q15c+du` means that the minimum map dimension (either map width or height) is 15 cm.